### DISCUSSION TITLE
Ffmpeg vs Ffmpeg-static vs Avconv

### DISCUSSION THREAD

**Discussion by Majoramari** [Androz2091/discord-player#881] [Q&A]
I'm just wondering what is the best one to use Ffmpeg, Ffmpeg-static or Avconv...

**EDIT:**
I think the Ffmpeg is the most stable one: #841

Maybe this would be interesting to you:
https://stackoverflow.com/questions/9477115/what-are-the-differences-and-similarities-between-ffmpeg-libav-and-avconv

**Comment by twlite** [comment]
ffmpeg-static downloads ffmpeg binaries
avconv is a fork of ffmpeg


use whatever you like.

**Reply by Majoramari** [reply]
I saw people saying ffmpeg-static is unstable I'm not sure why, so that made me wondering!
